app/views/customers/new.html.erb

```diff
@@ -4,6 +4,11 @@
       <div class="card card-default">
         <h1 id="card-text" class="card-header">Sign up</h1>
         <div class="card-body">
+          <script>
```

Comment:
I assume this is the cause, but I was unable to fill out this form because every time I selected a date, unselecting would clear it. In addition, it seems like there are two date pickers fighting for priority. Possibly a conflict with Firefox's internal date picker. Also, I don't think a date picker is really necessary to input MM/YY. A text field would suffice.